src/components/SearchFilterEntities.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { FormattedMessage, FormattedNumber } from 'react-intl';
+import { Tab2, Tabs2 } from '@blueprintjs/core';
+
```

Comment:
Ok, so all of this is about Schema, not entities (Schema applies to both entities and Documents)